cmake/Modules/opm-core-prereqs.cmake

```diff
@@ -37,6 +37,7 @@ set (opm-core_DEPS
 	# DUNE dependency
 	"dune-common"
 	"dune-istl"
+	"opm-common REQUIRED"
```

Comment:
On 08/09/15 15:09, André R. Brodtkorb wrote:
>
> In cmake/Modules/opm-core-prereqs.cmake 
> <https://github.com/OPM/opm-common/pull/35#discussion_r38922686>:
>
> > @@ -37,6 +37,7 @@ set (opm-core_DEPS
> >  	# DUNE dependency
> >  	"dune-common"
> >  	"dune-istl"
> > +	"opm-common REQUIRED"
>
> Still doesn't work for me, can you help me by checking that all 
> relevant changes were pushed and that it works at your end?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/OPM/opm-common/pull/35/files#r38922686>.
>
my bad, i had forgotten a hunk.

we cannot do REQUIRED on the opm-common dependency right now as 
opm-common does not produce a library and all dependencies are assumed 
to contain one.